src/index.js

```diff
@@ -1,38 +1,36 @@
 import './style.css';
 
-const todos = [
-  {
-    description: 'read a book',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'play videogames',
+import createTodo from './modules/createTodo.js';
+import removeTodo from './modules/delete.js';
+import editTodo from './modules/edit.js';
+
+const todoBtn = document.querySelector('#createTodo');
+const makeTodo = document.querySelector('#newTodo');
+let todosList = JSON.parse(localStorage.getItem('to-do'));
+
+if (todosList === null) {
+  todosList = [];
+}
+
+window.addEventListener('load', createTodo(todosList));
+// Add To-do functionality
+todoBtn.addEventListener('click', () => {
+  const { value } = makeTodo;
+  const todoObj = {
+    description: value,
     completed: false,
-    index: 1,
-  },
-  {
-    description: 'walk the dog',
-    completed: true,
-    index: 2,
-  },
-];
+    id: Math.random().toString(16).slice(2),
```

Comment:
According to the [project requirements:](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md)

![image](https://user-images.githubusercontent.com/57421823/154411186-fe63affa-ba8b-47fe-9b0c-bf407cf74da1.png)

Therefore: 

- [ ] Please, kindly for new tasks make the index property a numerical value that matches the new array length.  The indexes should be 1, 2, 3, 4... according to each item order on the list.
- [ ] When you delete a task, all the remaining items' indexes are updated, so they represent the current list order and are unique.

By judging what you already did, I can tell this should be no issue to you 💪
That way, we can move closer to project approval ✔️ 

Also:

- [ ] Please add a project description to your readme, and a "getting started" section.  Please, follow this [template](https://github.com/microverseinc/readme-template) closely as it's mandatory for nearly all the Microverse projects. 

All sections of the template are required, except for the "Tests" and "Live Demo" if not required by the project.
This information will serve you well for your future projects, and that way, your app will be well documented with a professional repo 💪